backend/src/main/java/com/google/models/Alert.java

```diff
@@ -87,18 +97,33 @@ public Entity toEntity() {
     return alertEntity;
   }
 
+  public static Alert createAlertWithoutId(Timestamp timestampDate, 
+      List<Anomaly> anomalies, StatusType status) {
+    return new Alert(timestampDate, anomalies, status, OptionalLong.empty());
+  }
+
   @SuppressWarnings("unchecked")
   public static Alert createAlertFromEntity(Entity alertEntity) {
-    List<EmbeddedEntity> listEE = (List<EmbeddedEntity>) alertEntity.getProperty(ANOMALIES_LIST_PROPERTY);
+    List<EmbeddedEntity> listEE = 
+        (List<EmbeddedEntity>) alertEntity.getProperty(ANOMALIES_LIST_PROPERTY);
 
     List<Anomaly> listAnomaly = new ArrayList<Anomaly>();
     if (listEE != null) {
-      listEE.forEach(embeddedAnomaly -> listAnomaly.add(Anomaly.createAnomalyFromEmbeddedEntity(embeddedAnomaly)));
+      listEE.forEach(
+        embeddedAnomaly -> listAnomaly.add(
+          Anomaly.createAnomalyFromEmbeddedEntity(embeddedAnomaly)
+        )
+      );
     }
 
-    return new Alert(new Timestamp((String) alertEntity.getProperty(Timestamp.TIMESTAMP_PROPERTY)), 
-        listAnomaly, 
-        StatusType.valueOf((String) alertEntity.getProperty(STATUS_PROPERTY)));
+    return new Alert(
+      new Timestamp(
+        (String) alertEntity.getProperty(Timestamp.TIMESTAMP_PROPERTY)
+      ), 
```

Comment:
optional: make this one line (I don't think it needs to be split among multiple lines. rather, it's more readable when it's in one line.)

Reply:
if it is one line, it exceeds 80 characters. thats why i split it up. do you have any better suggestion for splitting this up without going over 80 characters?